src/solvers/dgsem_tree/dg_3d_parabolic.jl

```diff
@@ -583,6 +593,298 @@ function calc_boundary_flux_by_direction_divergence!(surface_flux_values::Abstra
     return nothing
 end
 
+function prolong2mortars!(cache,
+                          flux_viscous::Tuple{AbstractArray, AbstractArray,
+                                              AbstractArray},
+                          mesh::TreeMesh{3},
+                          equations_parabolic::AbstractEquationsParabolic,
+                          mortar_l2::LobattoLegendreMortarL2,
+                          surface_integral, dg::DGSEM)
+    # temporary buffer for projections
+    @unpack fstar_tmp1_threaded = cache # CARE: Not sure if I have to do something with this (not present in 2D)
```

Comment:
```suggestion
    @unpack fstar_tmp1_threaded = cache
```
This is an auxiliary work array needed for the projections along a given face. It is used below in the `element_solutions_to_mortars!` calls. You do not need it in 2D because on a interface between 2D elements there is only a single coordinate direction along which the project is taken. In 3D we have faces, which contain a "xi" and "eta" direction so when the projection is performed we do one direction first and store it in a buffer then the projection along the other direction is performed to finish up the complete projection.

Reply:
Thanks for the detailed explanation!